docs/root/intro/arch_overview/upstream/load_balancing/panic_threshold.rst

```diff
@@ -79,8 +79,16 @@ priority.
 
 Panic mode can be disabled by setting the panic threshold to 0%.
 
-If all hosts become unhealthy normalized total health is 0%, and if the panic threshold is above 0%
-all traffic will be redirected to P=0.
+If the number of healthy hosts keeps decreasing, individual priority levels will start entering panic mode.
+Until at least one priority level is not in panic mode, load distribution is calculated as it is 
```

Comment:
I think flipping this sentence around would read a bit better: `Load distribution is calculated as describe above as long as there are priority levels not in panic mode`.

Reply:
Done.